Editor/AGS.Types/TranslationEntry.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AGS.Types
+{
+    public class TranslationEntry
+    {
+        public TranslationEntry()
+        {
+            metadata = new List<string>();
+        }
+
+        public string msgctxt { get; set; }
+        public string msgid { get; set; }
+        public string msgstr { get; set; }
```

Comment:
Please, make more comprehensible names, following the code style in AGS.Types.
These names do not need to follow the PO format keywords, as this is AGS translation entry, not PO's, we only save in PO.

E.g.:
```
string Context
string ID;
string String or Line;
etc
```